sigaex/src/main/java/br/gov/jfrj/siga/vraptor/ExMobilController.java

```diff
@@ -1169,4 +1171,34 @@ public void listar_docs_para_arquivar_corrente_lote(final String atendente, fina
 
 		}
 	}
+	
+	@Get("/app/expediente/mov/listar_docs_para_receber_em_lote")
+	public void listarDocsParaReceberEmLote(final String atendente, int offset) {
+		
+		int tamanho = dao().consultarQuantidadeDocsParaReceberEmLote(getTitular(), atendente);
+		List<ExMobil> itens = dao().consultarParaReceberEmLote(getTitular() ,atendente , offset, MAX_ITENS_PAGINA_CINQUENTA);
+		final List<ExMobil> l = new ArrayList<ExMobil>();
+			
+		for (ExMobil m : itens) {
```

Comment:
Avaliar a possibilidade de transcrever a verificação de isApensado para a NameQuery
Avaliar a possibilidade de chamar o método podeAcessarDocumento na jsp listarDocsParaReceberEmLote.jsp através da f:podeAcessarDocumento para reaproveitar a iteração do siga:paginador e remover esse laço for